fabric-networking-api-v1/src/client/java/net/fabricmc/fabric/api/client/networking/v1/ClientPlayNetworking.java

```diff
@@ -97,6 +163,9 @@ public static Set<Identifier> getGlobalReceivers() {
 	 * <p>For example, if you only register a receiver using this method when a {@linkplain ClientLoginNetworking#registerGlobalReceiver(Identifier, ClientLoginNetworking.LoginQueryRequestHandler)}
 	 * login query has been received, you should use {@link ClientPlayConnectionEvents#INIT} to register the channel handler.
 	 *
+	 * <p>For new code, {@link #registerReceiver(PacketType, PlayPacketHandler)}
+	 * should be used instead, as it is designed in a way that prevents thread safety issues.
```

Comment:
```suggestion
	 * should be preferred, as it is designed in a way that prevents thread safety issues.
```

Reply:
We prefer, not the readers; changed to "is preferred"